interfaces/cython/cantera/test/test_convert.py

```diff
@@ -478,26 +471,24 @@ def test_extra(self):
                      transport='gri30_tran.dat', output='gri30_extra',
                      extra='extra.yaml')
 
-        output = pjoin(self.test_work_dir, 'gri30_extra' + self.ext)
-        with open(output, 'rt', encoding="utf-8") as stream:
-            yml = yaml.safe_load(stream)
+        output = self.test_work_path / ("gri30_extra" + self.ext)
+        yml = utilities.load_yaml(output)
 
         desc = yml['description'].split('\n')[-1]
         self.assertEqual(desc, 'This is an alternative description.')
         for key in ['foo', 'bar']:
             self.assertIn(key, yml.keys())
-        # This test tests it can convert the SRI parameters when D or E equal to 0
-   
+
     def test_sri_zero(self):
+        # This test tests it can convert the SRI parameters when D or E equal to 0
```

Comment:
Can you make this a docstring (in triple quotes)? IIRC, pytest is able to display docstrings in certain modes.

Reply:
I am hesitant, as I have seen docstrings replace the actual stack information (i.e. what file / test). Also, I didn't really touch this line other than moving it to where it belongs.

Reply:
Marking as resolved as docstrings for unit tests is a larger discussion (that goes beyond the scope here: the comment belongs to a test that was not modified in this PR)